strictEqual.js

```diff
@@ -0,0 +1,5 @@
+function strictEqual(a, b) {
+  // eslint-disable-next-line eqeqeq
+  return ((a == b) && (typeof a == typeof b))
```

Comment:
you dont need the parentheses `return typeof num1 == typeof num2 && num1 == num2`